src/main/java/com/matera/cadastrocentral/client/Client.java

```diff
@@ -24,6 +25,9 @@
     @OneToMany(mappedBy = "client", cascade=CascadeType.ALL, orphanRemoval=true)
     @JsonManagedReference
     private List<IdentityDocumentEntity> identityDocumentEntityList;
+    @OneToMany(mappedBy = "clientId", cascade=CascadeType.ALL, orphanRemoval=true)
+    @JsonManagedReference
+    private List<Address> addressList;
 
```

Comment:
Maybe add Address at the constructor and in service because we wanted that when Client is called, it returns an object with all attributes (maritalStatus, documents, telephones and addresses). Is this happening here?